setup.py

```diff
@@ -76,10 +76,11 @@ def build_extension(self, ext):
 
 setup(
     name='hatchet',
-    version='0.4.9',
-    packages=['hatchet', 'hatchet.utils', 'hatchet.utils.solve', 'hatchet.bin', 'hatchet.data'],
+    packages=['hatchet', 'hatchet.utils', 'hatchet.utils.solve', 'hatchet.bin', 'hatchet.data', 'hatchet.resources'],
```

Comment:
I think we can put all data in `hatchet.data` (there's only a `sample.bbc` and `sample.seg` there currently), and save creating another submodule? Might be easier long term.